packages/std/src/iterator.rs

```diff
@@ -2,7 +2,8 @@ use crate::errors::StdError;
 use std::convert::TryFrom;
 
 /// A Key-Value pair, returned from our iterators
-pub type Pair<V = Vec<u8>> = (Vec<u8>, V);
+/// (since it is less common to use, and never used with default V, we place K second)
```

Comment:
We have Pair<T> used in many places throughout the code (especially in storage-plus)
I assume this will be the most common usage.

I was trying to avoid breakage. You can make the version with breakage and see which we like more.

Reply:
I'd also favour a version where the generic arguments are in the same order as the two fields. If this was a struct with two fields, the order would not matter that much. But here we have unnamed positional fields.

I see Ethan's point as well. We try to do a lot of different things with one symbol. In cosmwasm_std we don't need the generic arguments at all because all is binary. Then we build higher and higher and using the same type. I'm not surprised this ends up non-ideal. But at the end it is probably more beginner friendly to only have one type.

By the way, since this is a generic pair, we probably should rename the arguments to `A` and `B` for `.0` and `.1`.

Reply:
Please see #1106.

Reply:
> By the way, since this is a generic pair, we probably should rename the arguments to A and B for .0 and .1.

This will be super confusing. When I get it I usually write something like `|(k, v)| Ok(String::from_utf8(&k), from_slice(&v))`

Having A and B would make it more confusing.

This was previously named `KV` before being renamed to `Pair` and it is not intended to be generic, but return data from an Storage Iterator.